src/App.tsx

```diff
@@ -1,23 +1,44 @@
-import React from 'react';
-import logo from './logo.svg';
+import React, { FormEvent, useState } from 'react';
 import './App.css';
+import { useDispatch } from 'react-redux';
+import { useNavigate } from 'react-router-dom';
+import { authUser } from './store/userSlice';
+
+interface User {
```

Comment:
This should probably use the same type as expected by the `authUser` action (import it instead)